doc/release-notes.md

```diff
@@ -11,6 +11,15 @@ Option handling
   (provided that the wallet is enabled and pruning is disabled, and unless
   `-rescan=0` is specified explicitly).
 
+RPC Interface
+-------------
+
+- The default `minconf` value for `z_sendmany` is now 10 confirmations instead
+  of 1.
+- When spending notes using `z_sendmany`, Orchard anchors are now selected at
+  a depth of 10 blocks instead of at the chain tip. If the `minconf` value is
+  less than 10, `minconf` is used for the anchor selection depth instead.
```

Comment:
This isn't mentioning in the new `-anchorconfirmations` flag, is that intentional?